### DISCUSSION TITLE
PiP UI Aesthetic - Missing rounded corners

### DISCUSSION THREAD

**Discussion by wazeerc** [zen-browser/desktop#2769] [Ideas]
### Captchas

- [ ] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

The PiP window that pops up after switching tabs does not have rounded corners which is not really aesthetically pleasing

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.17

### What platform are you seeing the problem on?

Windows

**Comment by LeMoonStar** [comment]
Hey! For improvement requests, please use GitHub Discussions, not issues, as issues are meant for bugs.
